AsyncDisplayKit/ASImageNode.mm

```diff
@@ -181,7 +183,8 @@ - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
                                                     opaque:self.opaque
                                              contentsScale:self.contentsScaleForDisplay
                                            backgroundColor:self.backgroundColor
-                                               contentMode:self.contentMode];
+                                               contentMode:self.contentMode
```

Comment:
I'll volunteer to do an audit of the locking policies of ASTextNode and friends today. I'll file issues in the existing implementation if I find anything.